README.md

```diff
@@ -988,6 +988,15 @@ _This is a collection of posts from the community about the `hypersdk` and how t
 * [Launching Custom Tokens With HyperSDK By Avalanche](https://pythontony.hashnode.dev/launching-custom-tokens-with-hypersdk-by-avalanche)
 * [Avalanche VMs deep-dive #1: HyperSDK/tokenvm](https://ashavax.hashnode.dev/avalanche-vms-deep-dive-1-hypersdktokenvm)
 
+## HyperSDK Projects 👀
+_This is a gallery of projects currently building on top of the HyperSDK
+
+* [NodeKit: Decentralizing The L2 Sequunecer on a Subnet](https://nodekit.gitbook.io/nodekit-documentation/)
+* [OracleVM: Providing OffChain Data to the Avalanche Ecosystem](https://github.com/bianyuanop/oraclevm)
+* [ShuttleVM: Trusted Execution Environment for Wasm Programs to run distributed workloads on a subnet]
```

Comment:
link? I think the requirement to list a project is to have the project open source or a minimum link to something.

Reply:
I think a link is a good minimum requirement (even if to a website instead of the code at first)